### ISSUE TITLE
propsal: make LoraWan_RGB independent of actual RGB usage

### ISSUE THREAD

**Issue by eikaramba** [HelTecAutomation/CubeCell-Arduino#205]
currently there is no way to deactivate the internal RGB status flashes without completely deactivating all RGB functionality (then you cannot use the rgb at all anymore in your code)

i propose to decouple this. the LoraWan_RGB setting should only enable/disable the internal RGB usages. then you can still use it in your code to indicate stuff. another variable can be used to completely disable the rgb (if that is even needed)

**Comment by gasagna** [comment]
I agree. I tried to set `LoraWan_RGB` to zero and develop custom code for the NeoPixels, but I have been unable to do this so far.
